package.json

```diff
@@ -87,8 +87,10 @@
     "parse-script-tags": "^0.1.6",
     "pretty-fast": "^0.2.3",
     "prop-types": "^15.6.0",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
+    "react": "16.4.1",
+    "react-aria-components":
+      "https://github.com/juanca/react-aria-components.git",
```

Comment:
I realize we may not want to pull in another 3rd-party dependency. This one in particular is intended to be lightweight, and @juanca and I may publish components from it individually, potentially reducing even further the dependency.

Also, the tab component from this library is used in production at the company we both work for, and is "stable".

Reply:
do you have time to do a call and discuss this tomorrow?

Reply:
Yes I do 👍